### PR TITLE
Drop --bootstrap-version flag

### PR DESCRIPTION
**Pull request by sergii-ssh**
Drop --bootstrap-version flag for envoy binary. Flag has been deprecated https://github.com/envoyproxy/envoy/pull/17724

- [ ] Configuration Infrastructure
- [ ] Docs
- [ x ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ x ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: istio.deps [hunk 1]
```diff
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "74393cf764c167b545f32eb895314e624186e5b6"
+    "lastStableSHA": "f5051e663f7a20e7b21b515dda1b21f2d291f559"
```
Comment (hzxuzhonghu) [inline istio.deps:7]: Please don't couple this upgrade

Comment (sergii-ssh) [reply istio.deps]: done

**Comment by hzxuzhonghu** [comment]
lgtm

**Pull request merged**